iphone/templates/module/swift/template/ios/hooks/ti.swiftsupport.js

```diff
@@ -12,13 +12,18 @@ exports.init = init;
 
 /**
  * Main entry point for our plugin which looks for the platform specific
- * plugin to invoke
+ * plugin to invoke.
+ *
+ * @param {Object} logger The logger instance.
+ * @param {Object} config The hook config.
+ * @param {Object}cli The Titanium CLI instance.
+ * @param {Object} appc The Appcelerator CLI instance.
  */
 function init(logger, config, cli, appc) {
```

Comment:
<!--
  0 failure: 
  1 warning:  iphone/templates/...
  
  
  DangerID: danger-id-default;
  File: iphone/templates/module/swift/template/ios/hooks/ti.swiftsupport.js;
  Line: 22;
-->

- :warning: iphone/templates/module/swift/template/ios/hooks/ti.swiftsupport.js line 22 – 'appc' is defined but never used. (no-unused-vars)